lib/util/js.js

```diff
@@ -80,5 +81,21 @@ module.exports = {
       isFinite(value) &&
       Math.floor(value) === value &&
       value >= 0
+  },
+
+  /**
+   * Transform a 4-byte integer into a Buffer of length 4.
+   *
+   * @param {number} integer
+   * @return {Buffer}
+   */
+  integerAsBuffer: function integerAsBuffer (integer) {
```

Comment:
there is a built-in method in node for this i'm sure... fromUInt32BE/LE or something of the sort

Reply:
I've pushed an extra commit to use the built-in method.  Also double-checked the call sites to confirm they are passing in unsigned values i.e. there is no intent to pass in a negative integer.